File: code-examples/scala/collation/findOneAndUpdate-collation.scala
import org.mongodb.scala.model.Filters._
import org.mongodb.scala.model.Updates._
import org.mongodb.scala.model.Collation._
import org.mongodb.scala.MongoCollection
import org.bson.Document

val myColl: MongoCollection[Document] = // your mongo collection here

myColl.findOneAndUpdate(
  lt("first_name", "Gunter"),
  set("verified", true),
  Collation.builder().locale("de@collation=phonebook").build()
)

File: code-examples/php/collation/findOneAndUpdate-collation.php
<?php

require 'vendor/autoload.php';  // include composer's autoloader

$collection = (new MongoDB\Client)->test->myColl;

$updateResult = $collection->findOneAndUpdate(
    ['first_name' => ['$lt' => "Gunter"]],
    ['$set' => ['verified' => true]],
    ['collation' => ['locale' => 'de', 'strength' => 2]]
);

?>
